core/src/main/java/com/nerds/gamejam/ecs/WorldBuilder.java

```diff
@@ -5,24 +5,28 @@
 import com.artemis.WorldConfigurationBuilder;
 import com.artemis.managers.GroupManager;
 import com.artemis.managers.TagManager;
+import com.badlogic.gdx.graphics.Texture;
 import com.nerds.gamejam.GameJam;
 import com.nerds.gamejam.ecs.system.*;
 import com.nerds.gamejam.gameplay.planet.PlanetFactory;
 import com.nerds.gamejam.screen.MenuScreen;
+import com.nerds.gamejam.util.CachingTextureLoader;
 
 public class WorldBuilder {
 
     public static World build(GameJam game, MenuScreen menuScreen) {
+        CachingTextureLoader textureLoader = new CachingTextureLoader();
+        CameraSystem cameraSystem = new CameraSystem();
```

Comment:
This should be injected via @Wire to other systems

Reply:
I'm dubious about stuff like this, I know we're not at the test writing stage atm (and might never be) but is there a way for the @Wire stuff to work with testing?

Reply:
I don't think so. Artemis is pretty hard to test with anyway. You'll have same issue with world use

Reply:
You can use reflection magic to set the fields